contents/articles/roll-call-brett/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Roll call: Brett Langdon"
+author: brett-langdon
+date: 2015-06-21
+template: article.jade
+---
+
+This is the next part of our introductory series to look behind the curtain into the engineering team at [Underdog.io][].
+
+[Underdog.io]: https://underdog.io
+
+---
+
+<img alt="Photo of Brett" src="brett.jpg" style="width: auto; float: left; padding-right: 10px;" />
+
+#### Who are you and what do you do at Underdog.io?
+
+My name is Brett Langdon and I am senior software engineer at [Underdog.io][].
+
+Since we are still a small team, we all get to wear many hats and get our hands dirty with every aspect of our platform. Some days I am either writing Python/Node.js, writing HTML/CSS/JS, or staring at our [AWS][] management console/[Datadog][] dashboards.
+
+[AWS]: https://aws.amazon.com/
+[Datadog]: http://www.datadoghq.com/
+
+[Underdog.io]: https://underdog.io
+
+#### What hardware do you use?
+
+I use a 21" iMac and a WASD [CODE keyboard][] to help ensure the entire office knows when I am typing something.
+
+[CODE keyboard]: https://codekeyboards.com/
+
+#### What software do you use?
+
+I spend a good majority of my day in the terminal, so for productivity there I use [iTerm2][] as a terminal emulator, [oh-my-zsh][] to provide zsh plugins and themes, [homebrew][] for installing third party tools, and [tmux][] to organize my terminal workspace.
```

Comment:
Since you decided on slashes in the previous section, we might want to change the "zsh plugins and themes" to "plugins/themes"